cmd/geth/logtestcmd_active.go

```diff
@@ -42,6 +44,11 @@ This command is only meant for testing.
 // logTest is an entry point which spits out some logs. This is used by testing
 // to verify expected outputs
 func logTest(ctx *cli.Context) error {
+	// if log is set to output terminal format, reset root logger to clear field padding map
+	if !ctx.IsSet(debug.LogFormatFlag.Name) || ctx.String(debug.LogFormatFlag.Name) == "terminal" {
+		log.SetDefault(log.NewLogger(log.TerminalHandler(os.Stdout, false)))
```

Comment:
I changed it, so that internal/debug has access to the terminal format, but thne I had to make that public, and since it clashed with the constructur _and_ the constructor was named `TerminalHandler` instead of the more go-idiomatic `NewTerminalHandler`, I renamed the ctor and touched ~20+ files. Sigh. Anyway, feel free to undo my change if you don't like it, or think of something better 

Now I get no more errors on the `terminal`, but it appears the that the `logfmt` suddenly has timezone info in the timestamp, which apparently it didn't have before.

Reply:
Ah no it wasn't the tz that was new: 
```
old t=2023-10-20T12:56:08+0200
new t=2023-11-13T20:51:45.938+01:00
```
So... we should ignore millisecond level. I dunno, is the old (second-resolution) better than millisecond-resolution? For a machine-readable format???  Meh....

Reply:
Hm, actually two diffs: 
- milliseconds `.938`
- different format for tz, `+0200` vs `+01:00`.

Reply:
The built-in time field stopped being formatted according to our current time format at some point.

Reply:
crap we lost comma formatting for `*.Int`s somewhere along the way.

Reply:
No that's fine: https://github.com/ethereum/go-ethereum/pull/28187#issuecomment-1801291049